### PR TITLE
RF Distance

### PR DESCRIPTION
**Pull request by davidrich27**
- Adds methods for computing the RF distance.
- Adds rf distance logging to `larch-usher.cpp`
- Adds rf distance tests to `test/test-rf-distance.cpp`

### CODE REVIEW

File: include/larch/impl/merge/leaf_set_impl.hpp.orig
Comment (ognian-) [inline include/larch/impl/merge/leaf_set_impl.hpp.orig:1]: Should be removed

**Review by ognian- — APPROVED** [review]
Looks good! Just delete the .orig files.

File: include/larch/dag/connections.hpp
Comment (marybarker) [inline include/larch/dag/connections.hpp:1]: can we change these to ContiguousMap and ContiguousSets?

File: include/larch/impl/dag/connections_impl.hpp [hunk 1]
```diff
@@ -106,13 +106,13 @@ void FeatureMutableView<Connections, CRTP, Tag>::ClearConnections() const {
 }
 
 template <typename CRTP, typename Tag>
-ContiguousMap<ContiguousSet<NodeId>, ContiguousSet<NodeId>>
+std::map<std::set<NodeId>, std::set<NodeId>>
```
Comment (marybarker) [inline include/larch/impl/dag/connections_impl.hpp:109]: again, can this be changed to ContiguousMap?

**Review by marybarker — APPROVED** [review]

File: include/larch/dag/connections.hpp
Comment (davidrich27) [reply include/larch/dag/connections.hpp:1]: I tried changing this back to using ContiguousMaps and ContiguousSets.  Its still giving the same issues as before.

**Pull request merged**